apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -181,6 +207,32 @@ export async function calculatePoolStats({
     ),
   ]);
 
+  if (results[0].status === "fulfilled" && !results[0].value?.pool) {
+    throw new Error(
+      `No pool with ID ${poolId}(${network}) in range ${startAtTimestamp} - ${endAtTimestamp}`,
+    );
+  }
```

Comment:
if the pool returns no data in the subgraph it doesn't exist. This should not return an error but simply no result. It'd be interesting to find where those inexistent pool ids are coming from as well 🤔

Reply:
This should be non-issue since we're going to start getting pools id also from v2 API